labs/SizerBase/samples/SizerBase.Samples/SizerBase.Samples.csproj

```diff
@@ -14,6 +14,14 @@
     <RootNamespace>SizerBaseExperiment.Samples</RootNamespace>
     <AssemblyName>SizerBaseExperiment.Samples</AssemblyName>
   </PropertyGroup>
+
+  <ItemGroup>
+    <Content Remove="GridSplitter.md" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Remove="GridSplitter.md" />
+  </ItemGroup>
```

Comment:
We shouldn't need these here. Just need to unstage/discard before commit usually as VS is pretty aggressive about adding things here sometimes.